contrib/depends/packages/hidapi.mk

```diff
@@ -6,7 +6,7 @@ $(package)_sha256_hash=3c147200bf48a04c1e927cd81589c5ddceff61e6dac137a605f6ac979
 $(package)_linux_dependencies=libusb eudev
 
 define $(package)_set_vars
-$(package)_config_opts=--enable-static --disable-shared
+$(package)_config_opts=--enable-static --disable-shared --with-pic
```

Comment:
Can you make this $(package)_config_opts_linux=--with-pic ?

Reply:
Done.